baremetal_network_provisioning/db/bm_nw_provision_models.py

```diff
@@ -80,7 +80,7 @@ class BNPPhysicalSwitch(model_base.BASEV2, models_v2.HasId):
     priv_protocol = sa.Column(sa.String(16), nullable=True)
     priv_key = sa.Column(sa.String(255), nullable=True)
     security_level = sa.Column(sa.String(16), nullable=True)
-    __table_args__ = (sa.PrimaryKeyConstraint('id'),)
+    __table_args__ = (sa.PrimaryKeyConstraint('id', 'ip_address'),)
 
```

Comment:
need to make same changes in db/migration/alembic_migrations/versions/3297cd3f2323_bm_nw_provision_models.py also.